core/husarnet/husarnet_manager.cpp

```diff
@@ -479,10 +483,15 @@ void HusarnetManager::startWebsetup()
 void HusarnetManager::startHTTPServer()
 {
 #ifdef HTTP_CONTROL_API
-  auto server = new ApiServer(this);
+  std::string dashboardApiAddress{};
+  auto dashboardApiAddresses = getDashboardApiAddresses();
+  if(!dashboardApiAddresses.empty() && dashboardApiAddresses[0].isFC94()) {
```

Comment:
IMHO move this check logic to the `DashboardApiProxy` class, as at the end of the day it's the one knowing how and who to communicate with (and make the manager only pass data around, without too much logic)